### DISCUSSION TITLE
How to do a "all selected" with multiselect?

### DISCUSSION THREAD

**Discussion by 4FunAndProfit** [mantinedev/mantine#7069] [Q&A]
Hello !

I tried to change behavior of multiselect in order to not show all options when all are selected but rather show a text like "all selected", 

But i can't find a way to do it, 

I don't know if I'm the one who's overslept, but has anyone ever managed to do this or had any idea how to do it? 

Thanks a lot lot in advance :)

**Comment by rtivital** [comment]
You can do that with Combobox component

**Comment by 4FunAndProfit** [comment]
@rtivital thanks a lot ! did not tried with combobox directly, and effectively it works very easily :D
